src/components/ec-full-screen-overlay/ec-full-screen-overlay.vue

```diff
@@ -80,6 +80,14 @@ const { deactivate } = useFocusTrap(overlayContent, {
   allowOutsideClick: true,
 });
 
+function getFocusTrapContainer() {
+  return overlayContent.value;
+}
+
+defineExpose({
+  getFocusTrapContainer,
+});
```

Comment:
Can't find where we use it

Reply:
This will be used in EBO, where we will get the container in the `Ebo-full-screen-overlay.vue` file

Reply:
I see you added test, thank you :)